### PR TITLE
feat: playwright test migration

### PR DESCRIPTION
**Pull request by klizter**
Moving from [protractor](https://www.protractortest.org/#/) to [playwright](https://playwright.dev/).

Migration progress:
- [x] core-datepicker
- [x] core-dialog
- [x] core-progress
- [x] core-scroll
- [x] core-suggest
- [x] core-tabs
- [x] core-toggle

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+export default defineConfig({
+  testDir: './packages',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 1 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
```
Comment (klizter) [inline playwright.config.ts:12]: test flere workers i actions

File: playwright.config.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] }
+    },
+
+    {
+      name: 'webkit',
+      use: { ...devices['Desktop Safari'] }
+    }
```
Comment (klizter) [inline playwright.config.ts:42]: Add browsers:
1. Microsoft Edge
2. IE (?)

File: package.json [hunk 3]
```diff
@@ -20,50 +19,35 @@
     "publish:major": "node -r esm bin/index.js --publish=major",
     "start": "rollup --config --watch",
     "static-publish": "npm run build && static-publish --directory=packages --account=nrk-core --latest --major",
-    "test": "npm run build && npm run protractor && npm run lint",
+    "test": "npm run build && npx playwright test && npm run lint",
```
Comment (klizter) [inline package.json:22]: lint -> bygg -> test

File: jsconfig.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
```
Comment (klizter) [inline jsconfig.json:2]: needed?

**Comment by skjalgepalg** [comment]
Seven tests are presently skipped, and look to be annotated with `FIXME`:
```sh 
Running 333 tests using 4 workers
  7 skipped
  326 passed (41.4s)
```
Should either be fixed or pruned as deemed appropriate

**Review by skjalgepalg — APPROVED** [review]
This looks like a pretty solid foundation for now. Further tweaks can be moved to separate issues so we don't drag the scope of this issue out indefinitely.

**Pull request merged**